src/common/ios/iOSPlatform.ts

```diff
@@ -183,8 +183,11 @@ export class IOSPlatform extends GeneralMobilePlatform {
     }
 
     private generateSuccessPatterns(): Q.Promise<string[]> {
-        return this.getBundleId().then(bundleId =>
-            IOSPlatform.RUN_IOS_SUCCESS_PATTERNS.concat([`Launching ${bundleId}\n${bundleId}: `]));
+        return this.targetType === "device" ?
```

Comment:
Maybe check with `IOSPlatform.deviceString`is better.

Reply:
Otherwise LGTM

Reply:
Yup, fixed